tests/run-build.sh

```diff
@@ -8,6 +8,8 @@ if [ -f /opt/change-go-version.sh ]; then
     change-go-version 1.4
 fi
 
+SRC_CHANGES=$(git diff-tree --no-commit-id --name-only -r HEAD..origin/master | grep -vE "^Documentation/" | wc -l)
```

Comment:
I was thinking of adding another check for doc changes later, similar to 
```
DOC_CHANGES=$(git diff-tree --no-commit-id --name-only -r HEAD..origin/master | grep -E "^Documentation/" | wc -l)
```
and then have another if clause that handles documentation changes.